packages/libluajit/build.sh

```diff
@@ -2,79 +2,52 @@ TERMUX_PKG_HOMEPAGE=https://luajit.org/
 TERMUX_PKG_DESCRIPTION="Just-In-Time Compiler for Lua"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1:2.1.1763148144"
+TERMUX_PKG_VERSION="1:2.1.1763148144+g5c8cee3"
 TERMUX_PKG_SRCURL=git+https://github.com/LuaJIT/LuaJIT.git
 TERMUX_PKG_GIT_BRANCH=v${TERMUX_PKG_VERSION:2:3}
-TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_BREAKS="libluajit-dev"
 TERMUX_PKG_REPLACES="libluajit-dev"
 TERMUX_PKG_EXTRA_MAKE_ARGS="amalg PREFIX=$TERMUX_PREFIX"
 TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_UPDATE_TAG_TYPE=newest-tag
 
 termux_pkg_auto_update() {
-	local current_version="${TERMUX_PKG_VERSION#*:}"
-	local response latest_version unix_timestamp_latest
-	local api_url='https://archlinux.org/packages/search/json/?name=luajit'
-	# Get the latest version from Arch Linux's API.
-	# Since this project doesn't do release tags,
-	# Repology doesn't return a meaningfully interpretable latest version.
-	# We should also identify ourselves via 'User-Agent' as a courtesy.
-	response="$(curl -s \
-		-H 'User-Agent: Termux update checker 1.0 (github.com/termux/termux-packages)' \
-		"${api_url}"
+	local latest_version current_version="${TERMUX_PKG_VERSION#*:}"
+	latest_version="$(
+		TERMUX_PKG_SRCURL="https://gitlab.archlinux.org/archlinux/packaging/packages/luajit" \
+		termux_gitlab_api_get_tag
```

Comment:
We can grab the latest tag from the Arch GitLab instead of going though the `archlinux.org/packages` API.
Should be less prone to unknown failure modes since this is the exact same code used for all Gitlab instances.